### DISCUSSION TITLE
Automatic zoom and bound customization with map.fitbound

### DISCUSSION THREAD

**Discussion by alessandrorubio** [inocan-group/vue3-google-map#236] [Q&A] [answered]
Is it there a method/prop that allows to use map.fitbound from google maps api?

**Comment by HusamElbashir** [comment, marked as answer]
Hi @alessandrorubio 

We don't expose that functionality through our API. You'll have to use the native API instead to manually invoke `map.fitBounds`. Refer to the [advanced usage](https://github.com/inocan-group/vue3-google-map#advanced-usage) section in the docs for some usage patterns.
